### PR TITLE
Consolidating guidance for text editors

### PR DESCRIPTION
**Pull request by vickytnz**
Building on earlier notes to remove mention to Atom and building on it. 

Done in this ticket:

* removed the 1 remaining reference to Atom because it is now not actively in development
* removed 1 mention to Notepad++ for Windows as it was inconsistently recommended between the install guidance and building a basic prototype
* documented and created a consistent way to introduce Visual Studio Code (full name and then shortcut), and describe its benefits (keeping a particular note from the GOV.UK Prototype Kit guidance about accessibility) 

> You'll also need a code editor. Any HTML text editor will do, but we recommend <a href="https://code.visualstudio.com">Visual Studio Code</a> (also known as VS Code). This is because it is free, fairly accessible, and has lots of useful features.

| Before | After | 
| ---  | ---- |
| windows - choose text editor, mentions visual studio code as free | windows - choose text editor, mentions visual studio code with other name and changes |
| mac - choose text editor, mentions visual studio code as free |  mac - choose text editor, mentions visual studio code with other name and changes | 
| start make prototype with both VS Code and Notepad++ as options | start make prototype with Visual Studio Code as only option|

I also found an orphan page ['requirements'](https://prototype-kit.service-manual.nhs.uk/install/requirements) - i have changed it for now but will make a separate ticket to remove it

|  Before | After |
| --- | ---- |
| requirements page mentioning atom |  requirements page mentioning vs code |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,20 +27,36 @@ Title case.
...
-Use ‘NHS.UK’ for products that are for delivery teams and not the public (find out more on the [NHS health writing A-Z](https://service-manual.nhs.uk/content/a-to-z-of-nhs-health-writing#N))This includes the NHS.UK prototype kit, NHS.UK design system, NHS.UK service manual or NHS.UK website.
+
+We are changing how we talk about tools for delivery teams. These used to be called 'NHS.UK' products but we are now removing the '.UK'. This means that we talk about the:
+
+* NHS frontend (though this is still sometimes called 'NHS.UK frontend')
+* NHS digital service manual
+* NHS design system
+* NHS prototype kit
+
+> We are changing how we talk about the Service M
```
Comment (frankieroberto) [inline README.md:38]: Service Manual?

**Review by frankieroberto — COMMENTED** [review]
I wonder if we need “fairly accessible”? Sounds a bit of a weasel word phrase and might be hard to assess long-term?

File: README.md (hunk 1 above)
Comment (vickytnz) [reply README.md]: whoops that was a fragment that i didn't realise was still there! removed

**Comment by vickytnz** [comment]
> You'll also need a code editor. Any HTML text editor will do, but we recommend <a href="https://code.visualstudio.com">Visual Studio Code</a> (also known as VS Code). This is because it is free and has lots of useful features.

**Review by frankieroberto — APPROVED** [review]

**Pull request merged**